src/System.Private.ServiceModel/src/Internals/WcfEventSource.cs

```diff
@@ -2629,6 +2661,10 @@ public class EventIds
             public const int ThrowingEtwExceptionVerbose = 57409;
             public const int ThrowingEtwException = 57410;
             public const int HttpHandlerPickedForUrl = 62326;
+            public const int DummyAdminEventId = 65530;
+            public const int DummyOperationalEventId = 65531;
+            public const int DummyAnalyticEventId = 65532;
+            public const int DummyDebugEventId = 65533;
```

Comment:
Note that EventSource works by making a big array that is index by event ID.  This array is as big as the biggest event number used.  Thus it is best if you DON'T make big numbers.   Now the other numbers are what they are and for compat we have to live with that, but we  don't need to make it worse here.   From what I could tell there were plenty of IDs not being used less than 57K, so you should use something that does not make the range bigger.